app/routes/surveys/components/Submissions/Results.tsx

```diff
@@ -75,21 +78,24 @@ const Results = ({
   numberOfSubmissions,
 }: Props) => {
   const dispatch = useAppDispatch();
+  const event = useAppSelector((state) =>
+    selectEventById(state, { eventId: survey.event }),
+  ) as EventForSurvey;
```

Comment:
```suggestion
  const event = useAppSelector((state) =>
    selectEventById<EventForSurvey>(state, survey.event),
  );
```

Do you still expect an object as param?

Reply:
nvm this is for the event - that's not refactored yet